pegasus/sites.v3/code.org/public/printcertificates.haml

```diff
@@ -68,6 +72,6 @@ theme: none
       .page-break
     -name.chomp!
     -encoded = Base64.urlsafe_encode64(name)
-    %img{:src=>"/api/hour/certificate64/#{course}/#{encoded}.jpg", :width=>"100%"}
+    %img{:src=>"/api/hour/certificate64/#{script}/#{encoded}.jpg", :width=>"100%"}
     -count += 1
-  -slog(tag: 'print_certificates', course: course, count: count)
+  -slog(tag: 'print_certificates', course: script, count: count)
```

Comment:
Another thing I didn't change because I'm not exactly sure how it is used - we still slog as `course`